Parsers/ASimAuthentication/Tests/Microsoft_Github_Athentication_DataTest.csv

```diff
@@ -0,0 +1,2 @@
+﻿Result
```

Comment:
Authentication test not needed. remove this file

Reply:
Done